content/courses/onchain-development/local-setup.md

```diff
@@ -81,32 +105,35 @@ anchor test
 ```
 
 **The `anchor test` command should complete with no errors or warnings**.
-However you may encounter issues, and we'll fix them below:
 
-##### `package `solana-program v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
+**However you may encounter issues, and we'll fix them below:**
+
+- `package `solana-program
+  v1.18.12` cannot be built because it requires rustc 1.75.0 or newer` error
 
+This error is due to incompatible versions of `solana-program` and `solana-cli`.
 Run `cargo add solana-program@"=1.18.x"`, where `x` matches your version of
 `solana-cli`. Then re-run `anchor test`.
 
-##### `Error: Unable to read keypair file`
+- `Error: Unable to read keypair file`
```

Comment:
Please use headings (I think `####` not 5) that way we can deep link individual sections.